### PR TITLE
feat: show lucky winners

### PR DESCRIPTION
**Pull request by nguyenhuudungz**

### CODE REVIEW

File: src/pages/Campaign/LeaderboardLayout.tsx [hunk 1]
```diff
@@ -22,14 +22,21 @@ import {
 } from 'state/campaigns/hooks'
 import InfoHelper from 'components/InfoHelper'
 import { CampaignState } from 'state/campaigns/actions'
+import BigNumberJS from 'bignumber.js'
```
Comment (viet-nv) [inline src/pages/Campaign/LeaderboardLayout.tsx:25]: we also have bignumber from ether or JSBI, why do we need bignumber.js?

File: src/pages/Campaign/index.tsx [hunk 2]
```diff
@@ -219,9 +219,35 @@ export default function Campaign() {
...
         dispatch(setSelectedCampaign({ campaign: { ...selectedCampaign, status: 'Ongoing' } }))
       }
       if (selectedCampaign && selectedCampaign.status === 'Ongoing' && selectedCampaign.endTime < now + 1000) {
+        dispatch(
+          setCampaignData({
+            campaigns: campaigns.map(campaign => {
+              if (campaign.id === selectedCampaign.id) {
+                return {
+                  ...campaign,
+                  status: 'Ended',
```
Comment (nguyenhoaidanh) [inline src/pages/Campaign/index.tsx:244]: I think u can write a constant for that status, I saw many statuses in your code.

Comment (nguyenhuudungz) [reply src/pages/Campaign/index.tsx:244]: Sure, will refactor soon in other PR. This PR is urgent for release.

File: src/pages/Campaign/LeaderboardLayout.tsx (hunk 1 above)
Comment (nguyenhuudungz) [reply src/pages/Campaign/LeaderboardLayout.tsx:25]: Ok.

**Review by viet-nv — APPROVED** [review]

**Pull request merged**